.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "storage"]
 	path = storage
 	url = https://github.com/Azure/azure-storage-go
+	branch = v0.1.0
```

Comment:
Oh, lol, this is not glide. But still, the question is the same! Cannot it be tag instead of branch?